External/AvalonTools/AvalonToolkit_1.1_beta_patch.txt

```diff
@@ -0,0 +1,165 @@
+*** ./SourceDistribution/common/reaccsio.c.orig	2016-01-31 19:46:03.709178428 +0000
```

Comment:
The most recent version of the Avalon toolkit is 1.2, if the patch is still needed, it should apply to that.